### PR TITLE
Fixes for Houdini 17 

### PR DESCRIPTION
**Pull request by Idclip**
Based off Version 17.0.336

### CODE REVIEW

File: openvdb_houdini/houdini/SOP_OpenVDB_Rasterize_Points.cc [hunk 1]
```diff
@@ -3397,7 +3397,8 @@ SOP_OpenVDB_Rasterize_Points::cookVDBSop(OP_Context& context)
             // Setup VEX context
 
 #if UT_MAJOR_VERSION_INT >= 17
-            OP_Caller caller(this, DEP_ContextOptionsReadHandle{});
+            OP_Caller caller(this, context.getContextOptionsStack(),
+                     context.getContextOptions());
```
Comment (Idclip) [inline openvdb_houdini/houdini/SOP_OpenVDB_Rasterize_Points.cc]: Looks like this was updated once, unsure if this was for a previous or newer version than 17.0.336?

Comment (OpenVDB-DevTeam) [reply openvdb_houdini/houdini/SOP_OpenVDB_Rasterize_Points.cc]: It was for an earlier version, probably 17.0.140.

**Comment by zxiiro** [comment]
recheck

File: openvdb_houdini/houdini/SOP_OpenVDB_Rasterize_Points.cc (hunk 1 above)
Comment (jmlait) [reply openvdb_houdini/houdini/SOP_OpenVDB_Rasterize_Points.cc]: New code matches the current version in Houdini; apologies for not having sent it back as a PR ourselves.

**Comment by danrbailey** [comment]
Merging this PR as discussed in the TSC meeting yesterday.

**Pull request merged**